doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_custom_pcb.rst

```diff
@@ -39,7 +39,7 @@ The nRF54H20 DK uses multiple board files for its configuration:
 * `Risc-V cores configuration files`_
 
 You can use these files as a starting point for configuring your own custom board.
-The easiest way to do so, when creating a :ref:`Zephyr repository application <zephyr:zephyr-repo-app>`, is to create a copy of these folders under :file:`sdk-nrf-next/boards/arm/your_custom_board_name` (for the ARM configuration files) and :file:`sdk-nrf-next/boards/riscv/your_custom_board_name` (for the Risc-V configuration files), respectively.
+The easiest way to do so, when creating a :ref:`Zephyr repository application <zephyr:zephyr-repo-app>`, is to create a copy of these folders under :file:`sdk-zephyr/boards/nordic/your_custom_board_name`.
```

Comment:
```suggestion
The easiest way to do so, when creating a :ref:`Zephyr repository application <zephyr:zephyr-repo-app>`, is to create a copy of these folders under :file:`sdk-zephyr/boards/your_vendor_prefix/your_custom_board_name`.
```

Reply:
Shouldn't this be conformant with HWMv2 and link to https://docs.nordicsemi.com/bundle/ncs-latest/page/zephyr/hardware/porting/board_porting.html#board-terminology or https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/app_dev/board_support/board_names.html ?

Reply:
Is the board target supposed to match the board's folder structure? @gmarull @greg-fer